feat: add telemetry (#2078)
pankod/refine: .changeset/tame-windows-speak.md

---
"@pankod/refine-core": minor
---
Add a simple and transparent telemetry module to collect usage statistics defined within a very limited scope.

> Tracking is completely safe and anonymous. It does not contain any personally identifiable Information and does not use cookies. Participation is optional and users can opt out easily.

Reviewer: ```suggestion
Tracking is completely safe and anonymous. It does not contain any personally identifiable information and does not use cookies. Participation is optional and users can opt-out easily.
```
also maybe we can use an entirely instead of a completely.

> You can check it here for [details](https://refine.dev/docs/guides-and-concepts/telemetry/telemetry/).

Reviewer: ```suggestion
For more information, you can check the [documentation](https://refine.dev/docs/guides-and-concepts/telemetry/telemetry/).
```

> refine added a simple and transparent telemetry module to collect usage statistics defined within a very limited scope.
> 
> Tracking is completely safe and anonymous. It does not contain any personally identifiable Information and does not use cookies. Participation is optional and users can opt out easily.
> 
> You can check here for [details](https://refine.dev/docs/guides-and-concepts/telemetry/telemetry/).

Reviewer 2: ```suggestion
Add a simple and transparent telemetry module to collect usage statistics defined within a very limited scope.

Tracking is completely safe and anonymous. It does not contain any personally identifiable Information and does not use cookies. Participation is optional and users can opt out easily.

You can check it here for [details](https://refine.dev/docs/guides-and-concepts/telemetry/telemetry/).
```

---
"@pankod/refine-core": minor
---

> 

Reviewer 3: ```suggestion
```
